package/resource_monitor/resource_monitor/src/resource_query.c

```diff
@@ -19,6 +19,7 @@
 
 #include "sbp.h"
 
+//#ifdef DEBUG_RESOURCE_QUERY
```

Comment:
can remove

Reply:
I usually leave these in so it's easy to enable debugging output, I agree it's less than ideal though